src/var/www/hsmm-pi/View/LocationSettings/edit.ctp

```diff
@@ -17,6 +17,11 @@
 
 <?php
 echo $this->Form->create('LocationSetting', array(
+  'inputDefaults' => array(
+    'div' => 'form-group',
+    'label' => array('class' => 'col col-md-3 control-label'),
+    'wrapInput' => 'col col-md-9',
```

Comment:
These `wrapInput`s are making it into the HTML as attributes on `<input>` tags, I don't imagine you want that. The option seems to be intended for a plugin you're not using.